doc/jcli/transaction.md

```diff
@@ -26,9 +26,12 @@ There are also functions to help decode and display the
 content information of a transaction:
 
 * `info`
-* `id` to get the **Transaction ID** of the transaction
+* `data-for-witness` get the data to sign from a given transaction
+* `fragment-id` get the **Fragment ID** from a transaction in *sealed* state
 * `to-message` to get the hexadecimal encoded message, ready to send with `cli rest message`
 
+**DEPRECATED**:
+* `id` to get the Transaction ID of the transaction (use `data-for-witness` instead)
```

Comment:
Here you say that `id` gives the `Transaction ID`. While actually it gives the `data to sign`, the `data-for-witness`.

```suggestion
* `id` to get the Data to sign the transaction (use `data-for-witness` instead)
```

Reply:
updated with the same info as `data-for-witness`